dev-docs/bidders/sovrn.md

```diff
@@ -22,3 +22,160 @@ fpd_supported: true
 |------------|----------|----------------------|------------|----------|
 | `tagid`    | required | The sovrn Ad Tag ID  | `'315045'` | `string` |
 | `bidfloor` | optional | Bid floor in dollars | `'0.04'`   | `string` |
+
+### Bid Params for video ads
+
```

Comment:
Please add a note indicating that these video parameters go in mediatype.video rather than your bidder-specific params.

Reply:
Sorry to be picky, but please move 'These params should be added to `mediatype.video`.' out of the heading level 3. That will make the formatting rather awkward.

Reply:
@bretg Is this what you were looking for?